utils.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/bytebot-chat/gateway-irc/model"
 	"github.com/go-redis/redis/v8"
 	"github.com/rs/zerolog/log"
-	"github.com/satori/go.uuid"
+	uuid "github.com/satori/go.uuid"
```

Comment:
Why did we need to add that package namespace here?

Reply:
The linter in VS Code did it... i'm not really sure why.

Reply:
Let's back it out until we have a reason.

Reply:
Sounds good. Will do that now.